runtime/src/accounts_background_service.rs

```diff
@@ -399,7 +441,18 @@ impl AccountsBackgroundService {
                                 // slots >= bank.slot()
                                 bank.force_flush_accounts_cache();
                             }
-                            bank.clean_accounts(true, false, None);
+                            let last_full_snapshot_slot = request_handler
+                                .snapshot_request_handler
+                                .as_ref()
+                                .map(|snapshot_request_handler| {
+                                    snapshot_utils::calculate_last_full_snapshot_slot(
```

Comment:
Where is this defined?

Reply:
Which in particular? If you're asking about `calculate_last_full_snapshot_slot()`, it's in `snapshot_utils.rs` here:
https://github.com/solana-labs/solana/blob/5985f4f0b9fa17cd00c078194ad4ed89c7465f31/runtime/src/snapshot_utils.rs#L1739-L1746

But I'm no longer using it in this implementation.

Reply:
Got it -- I did not realize it was hidden in the file not expanded. Sorry for the confusion

Reply:
No worries! Thanks for being detailed.